CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.3.0](https://github.com/tari-project/randomx-rs/compare/v1.3.0...v1.2.1) (2023-11-01)
```

Comment:
This link doesnt seem to show changes

This is the best I could find
https://github.com/tevador/RandomX/compare/901f8ef765e7c274852dcb4d477247fd6747a5b8...102f8acf90a7649ada410de5499a7ec62e49e1da